spec/pubid_iso/identifier_spec.rb

```diff
@@ -647,6 +647,33 @@ module Pubid::Iso
       it_behaves_like "converts pubid to pubid"
     end
 
+    context "ISO/R 93-3:1969" do
+      let(:pubid) { "ISO/R 93-3:1969" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 93/1-1963" do
+      let(:original) { "ISO/R 93/1-1963" }
+      let(:pubid) { "ISO/R 93-1-1963" }
```

Comment:
The official pubid is [ISO/R 93-1:1963](https://www.iso.org/standard/3752.html).

We will need to implement a Roman numeral conversion for the old style part numbers.